httpx/_dispatch/wsgi.py

```diff
@@ -7,6 +8,15 @@
 from .base import SyncDispatcher
 
 
+def _get_non_empty_chunk(body: typing.Iterable) -> typing.Iterable:
+    """Get a non-empty chunk from body."""
```

Comment:
I notice you reverted the name, but I still think the current "get chunk" naming is not conveying correctly what this function does… A "get chunk" function should return a `chunk`, but this returns a modified `body` iterable that skips any leading empty chunks.

So, maybe…?

```suggestion
def _skip_leading_empty_chunks(body: typing.Iterable) -> typing.Iterable:
```

Reply:
That was a regression on my part.  Happy to accept any name whatsoever.

Reply:
Sure, sorry if my wording sounded offensive if any way :) I meant to write "I notice the name was reverted".